rules/sprint-review-retro-email/rule.md

```diff
@@ -68,90 +70,94 @@ Attendees: *(Optional as they may be in the to and CC)*
 | 24112  | Integrate Business Logic to MVC app | Done      | 8          |
 | 24097  | Styling                             | Committed | 16         |
 
-**Figure: Sprint Backlog from {{Link to Sprint Backlog in Azure DevOps}}** 
+**Figure: Sprint Backlog from {{ LINK TO SPRINT BACKLOG }}**
 
 1. Sprint Burndown (a quick overview of the Sprint)
 
-![Figure: Sprint Burndown](burndown.JPG)
+   ![Figure: Sprint Burndown](burndown.JPG)
+
+1. Code Coverage (hopefully tests are increasing each Sprint)
+
+   {{ CODE COVERAGE }}
+
+1. Velocity *(Optional)*
+
+   {{ VELOCITY }}
 
-2. Code Coverage (hopefully tests are increasing each Sprint)
-   XXX
-3. Velocity *(Optional)*
-   XXX
-4. Burnup (for the release - the whole project, how are we tracking for the big picture?)
+1. Burnup (for the release - the whole project, how are we tracking for the big picture?)
```

Comment:
This is an old image and report.
1. Should we replace this with something newer like the Cumulative Flow Diagram" from Azure DevOps?

![CleanShot 2024-01-16 at 10 52 37](https://github.com/SSWConsulting/SSW.Rules.Content/assets/5680199/e71a0f48-a250-43dc-9bc2-1ce9bfb15187)
**Figure: Screenshot of the TimePro CTF from https://dev.azure.com/ssw2/SSW.TimeProDotNet/_boards/analytics/t/SSW.TimeProDotNet%20Team/Backlog%20items/cfd**

Reply:
image repalced.